charts/base/templates/pvc.yaml

```diff
@@ -1,16 +1,16 @@
-{{- if .Values.storage -}}
+{{- range .Values.storage }}
```

Comment:
please make the change backword compatible by supporting both old single storage and new multiple storage cases, here we have example how you can do it: https://github.com/dasmeta/helm/blob/main/charts/base/templates/deployment.yaml#L56